VirtualReality/MRML/vtkMRMLVirtualRealityViewNode.cxx

```diff
@@ -78,7 +79,7 @@ void vtkMRMLVirtualRealityViewNode::WriteXML(ostream& of, int nIndent)
   vtkMRMLWriteXMLBooleanMacro(controllerTransformsUpdate, ControllerTransformsUpdate);
   vtkMRMLWriteXMLBooleanMacro(hmdTransformUpdate, HMDTransformUpdate);
   vtkMRMLWriteXMLBooleanMacro(controllerModelsVisible, ControllerModelsVisible);
-  vtkMRMLWriteXMLBooleanMacro(trackerReferenceModelsVisible, TrackerReferenceModelsVisible);
+  vtkMRMLWriteXMLBooleanMacro(trackerReferenceModelsVisible, LighthouseReferenceModelsVisible);
```

Comment:
Let's keep things consistent: update the attribute name, too.

    vtkMRMLWriteXMLBooleanMacro(lighthouseModelsVisible, LighthouseModelsVisible);

If we want backward compatibility then add reading of the old attribute name as well in ReadXMLAttributes.